src/App.js

```diff
@@ -3,6 +3,9 @@ import SearchBar from './components/layout/SearchBar'
 import Logs from './components/logs/Logs'
 import AddBtn from './components/layout/AddBtn'
 import AddLogModal from './components/logs/AddLogModal'
+import EditLogModal from './components/logs/EditLogModal'
```

Comment:
Again, I see you name folders like you'd have the same restrictions JavaScript has.